src/services/item/plugins/geolocation/ItemGeolocation.ts

```diff
@@ -0,0 +1,45 @@
+import {
+  BaseEntity,
+  Column,
+  CreateDateColumn,
+  Entity,
+  JoinColumn,
+  OneToOne,
+  PrimaryGeneratedColumn,
+  Unique,
+} from 'typeorm';
+import { v4 } from 'uuid';
+
+import { Item } from '../../entities/Item';
+
+@Entity({ name: 'item_geolocation' })
+@Unique('item_geolocation_unique_item', ['item'])
+export class ItemGeolocation extends BaseEntity {
+  @PrimaryGeneratedColumn('uuid')
+  id: string = v4();
+
+  // inherit from parent
+  @OneToOne(() => Item, (item) => item.path, {
+    onUpdate: 'CASCADE',
+    onDelete: 'CASCADE',
+    nullable: false,
+  })
+  @JoinColumn({ referencedColumnName: 'path', name: 'item_path' })
+  item: Item;
+
+  @Column({ type: 'float' })
+  lat: number;
+
+  @Column({ type: 'float' })
+  lng: number;
+
+  // easier for display in frontend
+  @Column({ type: 'character varying', nullable: true, length: 4 })
```

Comment:
If I have understand correctly, the GeoRepository raise an exception if lat, long are invalid (if the country is not found). Is there a reason to set country to nullable ? It is possible using the backend to insert a null country ?